server/src/main/java/com/defold/extender/services/CocoaPodsService.java

```diff
@@ -391,7 +391,7 @@ private String createMainPodFile(List<File> podFiles, File jobDirectory, File wo
             List<String> lines = Files.readAllLines(podFile.getAbsoluteFile().toPath());
             for (String line : lines) {
                 if (line.startsWith("platform :")) {
-                    String version = line.replaceFirst("platform :ios|osx", "").replace(",", "").replace("'", "").trim();
+                    String version = line.replaceFirst("platform :ios|platform :osx", "").replace(",", "").replace("'", "").trim();
```

Comment:
I'm a bit confused by the change. I don't see how this change will make any difference? At least not with the example string given in the issue.
```java
public class Main {
  public static void main(String[] args) {
    String line = "platform :ios '10.13'";    
    String version = line.replaceFirst("platform :ios|osx", "").replace(",", "").replace("'", "").trim();
    System.out.println("version: " + version);    // version: 10.13
  }
}
```

Reply:
Ah, no, never mind me. I tested with the wrong string

```java
public class Main {
  public static void main(String[] args) {
    String line = "platform :osx '10.13'";     
    String version = line.replaceFirst("platform :ios|osx", "").replace(",", "").replace("'", "").trim();
    System.out.println("version: " + version);   // version: platform : 10.13
  }
}
```

The regex could be changed to `"platform :(ios|osx)"` but it doesn't really matter much.